### DISCUSSION TITLE
[Question][Hw2] Disabled test/dune for coverage

### DISCUSSION THREAD

**Discussion by ahn9807** [prosyslab-classroom/cs424-program-reasoning#278] [Q&A]
Name: Junho Ahn

I turned off a few tests for the dragon part so I could see how much of the code is covered without it. 
I've implemented four out of the five sub-projects of Homework 2 needed. 
Is it ok if I don't run the dragon coverage test for now to pass the overall coverage test?

(Currently I upload to the grade scope without checking dragon tests)

**Comment by ahn9807** [comment]
It looks like I didn't pass 8 tests, which will be dragon unit tests.

**Comment by goodtaeeun** [comment]
If you find that it is the only way, you may do so.

**Comment by goodtaeeun** [comment]
I will close this issue.
